src/EnergyPlus/WindowManager.cc

```diff
@@ -6070,7 +6070,7 @@ namespace WindowManager {
 			TInRadFr = TInRad * root_4( ( 1.0 + 0.5 * ProjCorrFrIn ) / ( 1.0 + ProjCorrFrIn ) );
 			FrameCon = SurfaceWindow( SurfNum ).FrameConductance;
 			HInRad = 0.5 * SurfaceWindow( SurfNum ).FrameEmis * sigma * pow_3( TInRadFr + SurfaceWindow( SurfNum ).FrameTempSurfIn + TKelvin );
-			HInConvFr = 0.0;
+			HInConvFr = HInConv;
```

Comment:
This doesn't seem to be a problem with frame projections. It is only a problem when there is no frame projection (i.e., the frame is flush with the window and the wall). 

@EnergyArchmage what are your concerns with the one-line fix I'm proposing here? It seems to me that setting the convection to zero for the calculation of the surface temperatures, and then using a non-zero value to calculate the heat gain to the zone is disturbingly inconsistent. I don't see how this will have negative impacts elsewhere.

At a minimum, we should make the calculation for frames and dividers consistent.